apps/api/src/persistence/repositories/__snapshots__/arango-data-exporter.spec.ts.snap

```diff
@@ -0,0 +1,6598 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Let's modify this to only snapshot the additional fixture collections and not the entire test data.

Reply:
done